diff -rupN obs-studio-20.0.1.old/libobs/CMakeLists.txt obs-studio-20.0.1/libobs/CMakeLists.txt
--- obs-studio-20.0.1.old/libobs/CMakeLists.txt	2017-08-11 17:52:09.000000000 +0200
+++ obs-studio-20.0.1/libobs/CMakeLists.txt	2017-10-08 21:34:28.541068421 +0200
@@ -11,6 +11,10 @@ if (NOT "${FFMPEG_AVCODEC_LIBRARIES}" ST
 	list(REMOVE_ITEM FFMPEG_LIBRARIES ${FFMPEG_AVCODEC_LIBRARIES})
 endif()
 
+find_package(XCB COMPONENTS XCB REQUIRED RANDR REQUIRED XINERAMA REQUIRED)
+
+find_package(X11 REQUIRED)
+
 if(UNIX)
 	find_package(DBus QUIET)
 	if (NOT APPLE)
@@ -425,6 +429,10 @@ target_link_libraries(libobs
 		${OBS_JANSSON_IMPORT}
 		${FFMPEG_LIBRARIES}
 		${ZLIB_LIBRARIES}
+		${XCB_LIBRARIES}
+		${X11_XCB_LIBRARIES}
+		${X11_LIBRARIES}
+		-lm -ldl
 	PUBLIC
 		${THREADS_LIBRARIES})
 
