diff -up obs-studio-30.1.0/plugins/obs-browser/CMakeLists.txt.omv~ obs-studio-30.1.0/plugins/obs-browser/CMakeLists.txt
diff -up obs-studio-30.1.0/plugins/obs-browser/browser-app.cpp.omv~ obs-studio-30.1.0/plugins/obs-browser/browser-app.cpp
--- obs-studio-30.1.0/plugins/obs-browser/browser-app.cpp.omv~	2024-03-13 14:05:43.435619075 +0100
+++ obs-studio-30.1.0/plugins/obs-browser/browser-app.cpp	2024-03-13 14:05:59.963852266 +0100
@@ -147,7 +147,7 @@ void BrowserApp::ExecuteJSFunction(CefRe
 	std::vector<CefString> names;
 	browser->GetFrameNames(names);
 	for (auto &name : names) {
-		CefRefPtr<CefFrame> frame = browser->GetFrame(name);
+		CefRefPtr<CefFrame> frame = browser->GetFrameByName(name);
 		CefRefPtr<CefV8Context> context = frame->GetV8Context();
 
 		context->Enter();
@@ -346,7 +346,7 @@ bool BrowserApp::OnProcessMessageReceive
 		std::vector<CefString> names;
 		browser->GetFrameNames(names);
 		for (auto &name : names) {
-			CefRefPtr<CefFrame> frame = browser->GetFrame(name);
+			CefRefPtr<CefFrame> frame = browser->GetFrameByName(name);
 			CefRefPtr<CefV8Context> context = frame->GetV8Context();
 
 			context->Enter();
