diff -uraN obs-studio-27.1.0/libobs/CMakeLists.txt omv-obs-studio-27.1.0/libobs/CMakeLists.txt
--- obs-studio-27.1.0/libobs/CMakeLists.txt	2021-09-27 18:42:01.000000000 +0200
+++ omv-obs-studio-27.1.0/libobs/CMakeLists.txt	2021-09-28 19:29:48.669079439 +0200
@@ -10,6 +10,9 @@
 if (NOT "${FFMPEG_AVCODEC_LIBRARIES}" STREQUAL "")
 	list(REMOVE_ITEM FFMPEG_LIBRARIES ${FFMPEG_AVCODEC_LIBRARIES})
 endif()
+find_package(XCB COMPONENTS XCB REQUIRED RANDR REQUIRED XINERAMA REQUIRED)
+
+find_package(X11 REQUIRED)
 
 if(DEBUG_FFMPEG_MUX)
 	add_definitions(-DSHOW_SUBPROCESSES)
@@ -551,6 +554,10 @@
 		${OBS_JANSSON_IMPORT}
 		${FFMPEG_LIBRARIES}
 		${ZLIB_LIBRARIES}
+		${XCB_LIBRARIES}
+		${X11_XCB_LIBRARIES}
+		${X11_LIBRARIES}
+		-lm -ldl
 		Threads::Threads)
 
 install_obs_core(libobs EXPORT LibObs)
