diff -up obs-studio-31.1.2/frontend/cmake/os-linux.cmake.omv~ obs-studio-31.1.2/frontend/cmake/os-linux.cmake
--- obs-studio-31.1.2/frontend/cmake/os-linux.cmake.omv~	2025-09-09 00:35:36.000867863 +0200
+++ obs-studio-31.1.2/frontend/cmake/os-linux.cmake	2025-09-09 00:35:41.754352977 +0200
@@ -3,7 +3,7 @@ target_compile_definitions(
   obs-studio
   PRIVATE OBS_INSTALL_PREFIX="${OBS_INSTALL_PREFIX}" $<$<BOOL:${ENABLE_PORTABLE_CONFIG}>:ENABLE_PORTABLE_CONFIG>
 )
-target_link_libraries(obs-studio PRIVATE Qt::GuiPrivate Qt::DBus)
+target_link_libraries(obs-studio PRIVATE Qt::Gui Qt::DBus)
 
 find_package(Libpci REQUIRED)
 target_link_libraries(obs-studio PRIVATE Libpci::pci)
diff -up obs-studio-31.1.2/frontend/plugins/decklink-output-ui/CMakeLists.txt.omv~ obs-studio-31.1.2/frontend/plugins/decklink-output-ui/CMakeLists.txt
--- obs-studio-31.1.2/frontend/plugins/decklink-output-ui/CMakeLists.txt.omv~	2025-09-09 00:36:03.716652380 +0200
+++ obs-studio-31.1.2/frontend/plugins/decklink-output-ui/CMakeLists.txt	2025-09-09 00:36:08.138687686 +0200
@@ -38,7 +38,6 @@ target_link_libraries(
     Qt::Widgets
     "$<$<PLATFORM_ID:Darwin>:$<LINK_LIBRARY:FRAMEWORK,Cocoa.framework>>"
     $<$<PLATFORM_ID:Linux,FreeBSD,OpenBSD>:X11::X11>
-    $<$<PLATFORM_ID:Linux,FreeBSD,OpenBSD>:Qt::GuiPrivate>
 )
 
 if(OS_WINDOWS)
diff -up obs-studio-31.1.2/frontend/plugins/frontend-tools/CMakeLists.txt.omv~ obs-studio-31.1.2/frontend/plugins/frontend-tools/CMakeLists.txt
--- obs-studio-31.1.2/frontend/plugins/frontend-tools/CMakeLists.txt.omv~	2025-09-09 00:34:49.479637456 +0200
+++ obs-studio-31.1.2/frontend/plugins/frontend-tools/CMakeLists.txt	2025-09-09 00:35:04.102905310 +0200
@@ -65,7 +65,6 @@ target_link_libraries(
     Qt::Widgets
     "$<$<PLATFORM_ID:Darwin>:$<LINK_LIBRARY:FRAMEWORK,Cocoa>>"
     $<$<PLATFORM_ID:Linux,FreeBSD,OpenBSD>:X11::X11>
-    $<$<PLATFORM_ID:Linux,FreeBSD,OpenBSD>:Qt::GuiPrivate>
 )
 
 add_subdirectory("${CMAKE_SOURCE_DIR}/shared/obs-scripting" "${CMAKE_BINARY_DIR}/shared/obs-scripting")
