diff -up obs-studio-30.1.0/plugins/obs-browser/CMakeLists.txt.omv~ obs-studio-30.1.0/plugins/obs-browser/CMakeLists.txt
diff -up obs-studio-30.1.0/plugins/obs-browser/browser-app.cpp.omv~ obs-studio-30.1.0/plugins/obs-browser/browser-app.cpp
--- obs-studio-30.1.0/plugins/obs-browser/browser-app.cpp.omv~	2024-03-13 14:05:43.435619075 +0100
+++ obs-studio-30.1.0/plugins/obs-browser/browser-app.cpp	2024-03-13 14:05:59.963852266 +0100
@@ -147,7 +147,7 @@ void BrowserApp::ExecuteJSFunction(CefRe
 	std::vector<CefString> names;
 	browser->GetFrameNames(names);
 	for (auto &name : names) {
-		CefRefPtr<CefFrame> frame = browser->GetFrame(name);
+		CefRefPtr<CefFrame> frame = browser->GetFrameByName(name);
 		CefRefPtr<CefV8Context> context = frame->GetV8Context();
 
 		context->Enter();
@@ -346,7 +346,7 @@ bool BrowserApp::OnProcessMessageReceive
 		std::vector<CefString> names;
 		browser->GetFrameNames(names);
 		for (auto &name : names) {
-			CefRefPtr<CefFrame> frame = browser->GetFrame(name);
+			CefRefPtr<CefFrame> frame = browser->GetFrameByName(name);
 			CefRefPtr<CefV8Context> context = frame->GetV8Context();
 
 			context->Enter();
diff -up obs-studio-30.1.0/plugins/obs-browser/cmake/os-linux.cmake.omv~ obs-studio-30.1.0/plugins/obs-browser/cmake/os-linux.cmake
--- obs-studio-30.1.0/plugins/obs-browser/cmake/os-linux.cmake.omv~	2024-03-13 14:09:18.382575463 +0100
+++ obs-studio-30.1.0/plugins/obs-browser/cmake/os-linux.cmake	2024-03-13 14:10:19.971396804 +0100
@@ -1,6 +1,6 @@
 find_package(X11 REQUIRED)
 
-target_link_libraries(obs-browser PRIVATE CEF::Wrapper CEF::Library X11::X11)
+target_link_libraries(obs-browser PRIVATE CEF::Library X11::X11)
 set_target_properties(obs-browser PROPERTIES BUILD_RPATH "$ORIGIN/" INSTALL_RPATH "$ORIGIN/")
 
 add_executable(browser-helper)
@@ -13,7 +13,7 @@ target_sources(
 target_include_directories(browser-helper PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/deps"
                                                   "${CMAKE_CURRENT_SOURCE_DIR}/obs-browser-page")
 
-target_link_libraries(browser-helper PRIVATE CEF::Wrapper CEF::Library)
+target_link_libraries(browser-helper PRIVATE CEF::Library)
 
 set(OBS_EXECUTABLE_DESTINATION "${OBS_PLUGIN_DESTINATION}")
 
